scripts/set-environment

```diff
@@ -77,8 +77,8 @@ CI_PROJECT_REPONAME=${CI_PROJECT_REPONAME:-$CIRCLE_PROJECT_REPONAME}
   echo 'export BEHAT_ADMIN_PASSWORD=$(openssl rand -base64 24)'
   echo 'export BEHAT_ADMIN_USERNAME=pantheon-ci-testing-$CI_BUILD_NUMBER'
   echo 'export BEHAT_ADMIN_EMAIL=no-reply+ci-$CI_BUILD_NUMBER@getpantheon.com'
-  echo 'export MULTIDEV_SITE_URL="https://$TERMINUS_ENV-$TERMINUS_SITE.pantheonsite.io/"'
-  echo 'export LIVE_SITE_URL="https://live-$TERMINUS_SITE.pantheonsite.io/"'
+  echo "export MULTIDEV_SITE_URL='https://${TERMINUS_ENV:-$DEFAULT_ENV}-${TERMINUS_SITE:-$DEFAULT_SITE}.pantheonsite.io/'"
```

Comment:
Maybe we should evaluate `TERMINUS_ENV=${TERMINUS_ENV:-$DEFAULT_ENV}` (& same for TERMINUS_SITE) above rather than having multiple expansions of it in the `$BASH_ENV` generation.

Reply:
Done!